Effisense/sensor-dashboard
# Container and sensor page. Delete container and sensor.

Closes: #51, #9, #24

## Checklist


## Changelog
Detailed view about one container 
- Type of container
- Size of container
- Note
- Button to delete container with modal
- Button to update metadata about container (fields in the db)
- List over sensors that belong to the container

Detailed view about one sensor
-  Type of container with link to page for that container
- Location
- Note
- Button to delete sensor with modal
- Button to update metadata about sensor (for instance type of container, note, etc...)

## How to test
- Make sure you have a sensor on the map on the dashboard and a container connected to this sensor
- Click on the see more button on the popup to see the sensorview
- Click on the container name to see the containerview
- If you delete the container this field will also be updated in the sensorview
- If a container has no sensors then the list will be empty (maybe a check should be added here)

Notes:
- Styling is not done
- Map in both views will be implemented in a new issue
